<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_scr_dish_language_switch_set</class>
 <widget class="QDialog" name="w_scr_dish_language_switch_set">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
      <item>
       <widget class="ftableview" name="tableView"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="btnlanguageswicth" name="pushButton_language">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>90</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>90</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="outer_stylesheet" stdset="0">
           <string>pushbutton_type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_language_clear">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>清除</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_ok">
       <property name="text">
        <string>替换</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_toolbar_button</class>
   <extends>QPushButton</extends>
   <header>lds_toolbar_button.h</header>
  </customwidget>
  <customwidget>
   <class>ftableview</class>
   <extends>QTableView</extends>
   <header>ftableview.h</header>
  </customwidget>
  <customwidget>
   <class>btnlanguageswicth</class>
   <extends>QPushButton</extends>
   <header>btnlanguageswicth.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
